docs/io.cozy.accounts.md

```diff
@@ -143,31 +135,15 @@ in the `metadata` key of the relationship item.
 
 ```json
 {
+  "account_type": "freemobile",
   "auth": {
-    "accountName": "freemobile",
-    "credentials_encrypted": "aaDtwWSWsdpbbbbbbbbbbbbbbOzp2pBEbXlZLWjiTzOGumGRomrF2LwlRn4Y8c=",
-    "folderPath": "/Administratif/Free Mobile",
+    "credentials_encrypted": "aaDtwWSWsdpbbbbbbbbbbbbbbOzp...n4Y8c=",
     "login": "000000000",
-    "namePath": "Free Mobile",
-    "password": "*******"
-  }
-}
-```
-
-What we aim for:
-
-```json
-{
-  "auth": {
-    "credentials_encrypted": "aaDtwWSWsdpbbbbbbbbbbbbbbOzp2pBEbXlZLWjiTzOGumGRomrF2LwlRn4Y8c=",
-    "login": "000000000",
-    "password": "**********"
+    "password": "*******" // only when setting it first time
```

Comment:
What does that mean?

Reply:
I don't see the meaning of // only when setting it first time too. If this is a reference to the encryption of the password, It actually depends on the token you use to read it. I think you should remove this comment.

Reply:
If credentiails_entrypted is present, password will not be present, right

Reply:
I remove the password fields from example.